lib/improver/tests/utilities/cube_manipulation/test_MergeCubes.py

```diff
@@ -74,9 +72,9 @@ def setUp(self):
         cube1.attributes["history"] = (
             "2017-01-18T08:59:53: StaGE Decoupler")
 
-        # set up a 2D cube with 4 hour forecast period
+        # set up a 3D cube with 4 hour forecast period
         cube2 = set_up_variable_cube(
-            data[1].copy(), standard_grid_metadata='uk_ens', time=time_point,
+            data.copy(), standard_grid_metadata='uk_ens', time=time_point,
```

Comment:
See comment below, but I think these Test__equalise_cubes tests can all go away with the _equalise_cubes function. The stuff being tested here is all done by other functions that should have their own tests (and if they don't, these tests can be employed for those functions).